src/Components/Articles.jsx

```diff
@@ -1,29 +1,50 @@
 import { useEffect, useState } from 'react';
 import { useParams } from "react-router-dom";
+//import { useSearchParams } from "react-router-dom";
 import { Link } from 'react-router-dom';
 import * as api from '../api';
 import ArticlePreviewCard from './ArticlePreviewCard';
 
 const Articles = () => {
 const [articles, setArticles] = useState([]);
 const [isLoading, setIsLoading] = useState(false);
-
 const { topic } = useParams();
+const [sortBy, setSortBy] = useState('created_at');
+
+/*const [searchParams, setSearchParams] = useSearchParams({});
+const sortBy = searchParams.get('sort_by');
+
+const setSortBy = option => {
+  const newParams = new URLSearchParams(searchParams);
+  newParams.set('sort_by', option);
+  setSearchParams(newParams);
+}*/
```

Comment:
remove commented out code. 
On a bigger project with others, you should never leave commented out code. 
Someone else won't delete it, as they'll presume its needed at some point. 
It just ends up hanging around, cluttering up the code.